tinyagentos/projects/task_store.py

```
@@ -441,6 +457,84 @@ async def release_task(self, task_id: str, releaser_id: str) -> bool:
+        a separate (racy) pre-read.
+        """
+        now = time.time()
+        guard = (
+            "status = 'open' AND claimed_by IS NULL"
+            if only_if_unclaimed
+            else "status NOT IN ('closed', 'cancelled', 'parked')"
+        )
+        cursor = await self._db.execute(
+            f"""UPDATE project_tasks
+                SET status = 'parked', updated_at = ?
+                WHERE id = ? AND {guard}""",
```

Ruff (0.16.3)

[error] Possible SQL injection vector through string-based query construction

(S608)